web/viewer.css

```diff
@@ -1923,22 +1887,12 @@ html[dir='rtl'] #documentPropertiesOverlay .row > * {
   .visibleSmallView {
     display: inherit;
   }
-  html[dir='ltr'] #outerContainer.sidebarMoving .outerCenter,
```

Comment:
If you don't mind explaining, what was the reason for removing the `#outerContainer.sidebarMoving .outerCenter` rules completely in this patch?